main/src/cgeo/geocaching/storage/DataStore.java

```diff
@@ -229,6 +237,12 @@ public String call(final Cursor cursor) {
             + "title text not null, "
             + "updated long not null"
             + "); ";
+    private static final String dbCreateCachesLists = ""
+            + "create table " + dbTableCachesLists + " ("
+            + "list_id integer not null, "
+            + "geocode text not null, "
+            + "primary key (list_id, geocode)"
```

Comment:
I'm totally not a database expert, so this might be wrong: I would expect that we should have 2 additional indexes on the list_id and on the geocode, because we know for sure that we have a query that ask for all caches of a single list, and that we have a query which asks for all lists a cache is in. Otherwise those always have to do a full table scan, right?